File: backend-v2/src/main/java/vn/io/vutiendat3601/shop/v2/address/AddressDetailDto.java
package vn.io.vutiendat3601.shop.v2.address;

public record AddressDetailDto(
    String code,
    Long provinceId,
    String provinceName,
    Long districtId,
    String districtName,
    Long wardId,
    String wardName,
    String street,
    String customerCode) {}

File: frontend-v2/src/app/domain/address/address-detail-dto.ts
export class AddressDetailDto {
  constructor(
    public code: string,
    public provinceId: number,
    public provinceName: string,
    public districtId: number,
    public districtName: string,
    public wardId: number,
    public wardName: string,
    public street: string,
    public customerCode: string
  ) {}
}
